apps/src/lib/util/firehose.js

```diff
@@ -0,0 +1,114 @@
+/** @file Provides clients to AWS Firehose, whose data is imported into AWS Redshift. */
+
+import AWS from 'aws-sdk';
+
+const IDENTITY_POOL_ID = 'us-east-1:fbfec393-0afb-4682-84d4-59ad04a302f4'
+const AWS_REGION = 'us-east-1'
+
+AWS.config.region = AWS_REGION;
```

Comment:
Yeah, I think that's perfectly fine. My use case for this will be on the signup page, where at least in theory I'd only be sending an event if the user switched the selection to "Teacher", so it could trim down the number of calls quite a bit. This may or may not be a concern, depending on whether we decide this is a potential cost-generator or not.

Re: async, I think it's fine for this PR if it's synchronous, though something I'm realizing is that in the original test snippet we were passing around, we had all of the real work inside a callback like 'AWS.config.credentials.get(', which is missing here (as well as in the branch that I shared). I'm not really sure if that's necessary, maybe the credential setup does happen automatically behind the scenes? In which case there's a potential timing or failure issue, I'd imagine.